app/src/main/res/layout/fragment_transaction_details.xml

```diff
@@ -59,68 +59,36 @@
-                android:visibility="gone"
-                tools:visibility="visible"
-                android:orientation="vertical">
-
-                <io.gnosis.safe.ui.transactions.details.view.TxConfirmationsView
-                    android:id="@+id/tx_confirmations"
-                    android:layout_width="match_parent"
-                    android:layout_height="wrap_content"
-                    android:layout_marginTop="16dp"/>
-
-                <View
-                    android:layout_width="match_parent"
-                    android:layout_height="1dp"
-                    android:background="@color/light_grey"/>
-
-                <io.gnosis.safe.ui.settings.view.SettingItem
-                    android:id="@+id/advanced"
-                    android:layout_width="match_parent"
-                    android:layout_height="@dimen/item_setting_openable_height"
-                    app:setting_name="@string/tx_details_advanced"
-                    app:setting_openable="true" />
-
-                <View
-                    android:layout_width="match_parent"
-                    android:layout_height="1dp"
-                    android:background="@color/light_grey"/>
-                
-                <io.gnosis.safe.ui.transactions.details.view.TimeInfoItem
-                    android:id="@+id/created"
-                    android:layout_width="match_parent"
-                    android:layout_height="wrap_content"
-                    app:time_info_name="@string/tx_details_created"/>
-
-                <View
-                    android:layout_width="match_parent"
-                    android:layout_height="1dp"
-                    android:background="@color/light_grey"/>
-
-                <io.gnosis.safe.ui.transactions.details.view.TimeInfoItem
-                    android:id="@+id/executed"
-                    android:layout_width="match_parent"
-                    android:layout_height="wrap_content"
-                    app:time_info_name="@string/tx_details_executed"/>
-
-                <View
-                    android:layout_width="match_parent"
-                    android:layout_height="1dp"
-                    android:background="@color/light_grey"/>
-
-                <TextView
-                    android:id="@+id/etherscan"
-                    style="@style/TextLink"
-                    android:layout_marginLeft="16dp"
-                    android:layout_width="wrap_content"
-                    android:layout_height="@dimen/item_setting_openable_height"
-                    android:drawableRight="@drawable/ic_link_green_24dp"
-                    android:drawablePadding="6dp"
-                    android:gravity="left|center_vertical"
-                    android:text="@string/tx_details_view_on_etherscan"/>
+                android:orientation="vertical"
+                tools:visibility="visible">
+            <!-- Put Transfer ViewStub here  -->
+            <ViewStub
```

Comment:
View use a different stub for each tx detail layout ... this means if we add in the future more layout we need to add more and more stubs, not sure if this is the best idea (btw I like the ViewStub approach ;) )

Reply:
I moved the common sections out of the ViewStubs.